snapshot/snapshotter.go

```diff
@@ -280,9 +280,7 @@ type Snapshotter interface {
 	// A committed snapshot will be created under name with the parent of the
 	// active snapshot.
 	//
-	// Commit may be called multiple times on the same key. Snapshots created
-	// in this manner will all reference the parent used to start the
-	// transaction.
+	// Key will no longer be available after successful Commit.
```

Comment:
Word this as an absolute, not something that had happened.

"After commit, the snapshot identified by key is removed."